grammar.js

```diff
@@ -214,6 +221,8 @@ module.exports = grammar({
       $._type
     ),
 
+    blank_identifier: $ => seq('_'),
```

Comment:
I think `identifier` will match `_`. Could we just parse `_` as an identifier here as well?

Reply:
@maxbrunsfeld 👍 this is updated in https://github.com/tree-sitter/tree-sitter-go/pull/8/commits/92193205b44e0352b4cc7c9dc76a6c38f4a361c3.